hedera-node/src/main/java/com/hedera/services/ServicesState.java

```diff
@@ -358,6 +362,7 @@ private void internalInit(
 			networkCtx().setStateVersion(CURRENT_VERSION);
 
 			metadata = new StateMetadata(app);
+			app.workingState().updateChildrenFrom(this);
```

Comment:
This is needed for `genesisInit` as well?

Reply:
`genesisInit` delegates to `internalInit` as well right

Reply:
yeah , I was wondering if we need to do that for `genesisInit` since it is going to be default children with no entities. But nvm understood.